### PR TITLE
ENH: new logo for pyproj

### LINKED ISSUE #700: pyproj logo
It would be nice for pyproj to have a logo.

PROJ Logo for reference: https://github.com/OSGeo/PROJ/pull/1659

Please share with anyone you think might be interested in coming up with some possibilities.

### PR DESCRIPTION
**Pull request by direvus**
The logo is based on the logo for PROJ, designed by Sylvain Beorchia.

The logo preserves the same "axis" symbol in place of the letter O as in
PROJ, but changes the font to Sigmar One and adds the letters P and Y in
the same colours as the Python logo snakes.

As the author of this logo I release it under the CC0 "no rights
reserved" license.

new pyproj logo

 - [x] Closes #700
 - [x] Fully documented, including `history.rst` for all changes and `api/*.rst` for new API

### CODE REVIEW

**Comment by snowman2** [comment]
Thanks @direvus! I like what you have done. What are your thoughts on the letters being closer together?

**Comment by direvus** [comment]
No objections @snowman2  I can prepare a version with less spacing between the letters.

The logo was based on the PROJ logo, which has a lot of space between the objects.  I actually already moved them closer together compared with PROJ, in the version I posted above.  A bit of spacing might help with readability at small logo dimensions, but I'll play around with it and see how it comes out.

**Comment by snowman2** [comment]
Thanks for being flexible 👍

> A bit of spacing might help with readability at small logo dimensions

That might be true. I will need to look at the current version with different dimensions later.

**Comment by direvus** [comment]
Hi @snowman2 what do you think of this variant?

logo

logo_48

**Comment by snowman2** [comment]
> what do you think of this variant?

I like it :+1:

**Comment by direvus** [comment]
Cool, I'll update my PR with the reduced spacing in a few minutes.  Here's an icon concept to go along with the design as well, let me know if you want this included in the PR:

16px
icon_16 

32px
icon_32

**Comment by snowman2** [comment]
@direvus I like the logo idea. Thoughts on using one of the python colors for the circle and a different one for the line going across instead of for filling in the center? Almost like the two python snakes in a different orientation.

**Comment by direvus** [comment]
Sure, there's two different options for that idea, depending on whether you want the "axis" behind or in front of the "globe".

32px, "axis" in front
icon_32b

32px, "axis" behind
icon_32c

Personally I like axis in front better out of those two.

**Comment by snowman2** [comment]
I like that. What about one side of the axis in front and one behind?

**Comment by direvus** [comment]
32px, axis threaded
icon_32d

16px, axis threaded
icon_16d

FWIW I think the axis in front version works better out of these 3 options. The detail of the axis going behind becomes kind of difficult to make out at 16px

**Comment by direvus** [comment]
To recap the current options for the icon:

A: icon_16

B:  icon_16b

C: icon_16c

D: icon_16d

If we end up going with any of B, C or D, I will adjust the design of the "O" symbol in the main logo to match.

**Comment by snowman2** [comment]
I agree with you that B looks better small. I vote for that one for the icon.

**Comment by direvus** [comment]
OK thanks for the feedback, for now I will add the B style icon, roll that design into the main logo, and update my PR.

**Comment by snowman2** [comment]
That looks great. Thanks 👍

File: docs/conf.py [hunk 1]
```diff
@@ -96,12 +96,16 @@
 # a list of builtin themes.
 #
 html_theme = "sphinx_rtd_theme"
+html_logo = 'media/logo.png'
```
Comment (snowman2) [inline docs/conf.py:99]: The linter check is complaining about wanting double quotes `"` for strings.

File: docs/media/icon.svg [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="64"
+   height="64"
+   viewBox="0 0 16.933333 16.933333"
+   version="1.1"
+   id="svg3821"
+   inkscape:version="0.92.4 (5da689c313, 2019-01-14)"
+   sodipodi:docname="icon.svg"
+   inkscape:export-filename="\\wsl$\Ubuntu-18.04\home\direvus\src\pyproj\docs\media\icon.png"
```
Comment (snowman2) [inline docs/media/icon.svg:17]: Possible to make this path relative?
```
inkscape:export-filename="docs\media\icon.png"
```

**Comment by direvus** [comment]
Thanks for that latest round of feedback, all those changes are in the branch now.

**Comment by snowman2** [comment]
@direvus thoughts about adding the logo to the README?

**Comment by direvus** [comment]
Sounds good, happy to make that change

**Comment by snowman2** [comment]
Thanks @direvus 👍. Any issues with a squash and merge?

**Comment by direvus** [comment]
No worries at all, squash is fine by me :)

**Comment by snowman2** [comment]
Thanks again! Should be live soon :rocket:

**Pull request merged**